programs/protocol_event/src/state/event.rs

```diff
@@ -0,0 +1,36 @@
+use anchor_lang::prelude::*;
+
+#[account]
+pub struct Event {
+    pub authority: Pubkey,
+    pub payer: Pubkey,
+
+    pub category: Category,
+    pub event_group: EventGroup,
+
+    pub active: bool,
+
+    pub slug: String, // event identifier e.g. LAFCvLAG@2021-08-28
+    pub name: String, // for display purposes e.g. Los Angeles Football Club vs. LA Galaxy
+
+    pub participants: Vec<Participant>,
+
+    pub expected_start_timestamp: i64,
+    pub actual_start_timestamp: Option<i64>,
+    pub actual_end_timestamp: Option<i64>,
+}
+
+pub struct Category {
+    pub id: String, // category identifier e.g. FOOTBALL
+    pub name: String, // for display purposes e.g Football
+}
+
+pub struct EventGroup {
```

Comment:
Is `Event` in `EventGroup` redundant? I think it probably makes sense here but just asking the question as I'm aware of all the places we have put redundant prefixes in the past that now are irritating.

Reply:
Yeah good point, no reason to have `Category` and then `EventGroup`, I will remove `Event` for now at least for consistency and see how it looks further down the line

Reply:
I'm not pushing either way, though I'll be pointing out anywhere we introduce potentially redundant prefixes going forward so we definitely think about them. We did have it on the original hierarchy proposal
```
Category (e.g., Sports, Politics, Entertainment)
\_Sub Category (e.g., Basketball)
   \_Class? (e.g., European Basketball or Women's Baskball)
      \_Event Group (e.g., NBA, WNBA, NCAAB)
        \_Event (Lakers v Bulls June 12 2023)
```

Reply:
If we want to store them in their own accounts further down the line it maybe does make sense to use `EventGroup` as it's a bit less ambiguous than `Group`, I do like `EventGroup` better here but I will have a ponder on it